LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any equipment obtaining a copy
```

Comment:
I can see that you have used **Find and Replace All** to do refactoring 😅.

Reply:
Hahaha opps. ok we will this line equipment back to person